gdk/static/config_schema.json

```diff
@@ -64,21 +64,58 @@
                                     "build_system"
                                 ]
                             },
-                            "if": {
-                                "properties": {
-                                    "build_system": {
-                                        "const": "custom"
+                            "allOf": [
```

Comment:
This change allow us to have multi conditional if statements which is now required to accept different options per build system type

https://json-schema.org/understanding-json-schema/reference/conditionals.html